config/firebaseApp.config.js

```diff
@@ -0,0 +1,10 @@
+export const firebaseConfig =
+{
+  apiKey: "AIzaSyBZc6TvB-DUnQf4zjjDgSgPxeoVk_SvNsE",
```

Comment:
@abhinavbansal19961996 Load all this values from .env file

Reply:
how to fetch it on client side from .env? @zeenfaizpy